Backend/DTOs/FavouriteDTO.cs

```diff
@@ -0,0 +1,8 @@
+namespace Backend.DTOs;
+
+public class FavouriteDTO
+{
+    public int Id {get;set;}
+    public string UserEmailOrUsername { get; set; } = null!;
+    public long MovieId { get; set; }
```

Comment:
If the endpoint is expecting userId and movieId, do we need username and email?

Reply:
@NoHop3 That id is the id of the favourite movie (ef core was complaining that there is no id). I can change the DTO so it has the uderId instead of the usermandOrEmail. Should I do the same in the Review endpoints?